docs/langref/relay/expressions.rst

```diff
@@ -0,0 +1,162 @@
+===========
```

Comment:
Let us move directly into docs/langref/relay_expr.rst

The future trend will just make everything as a one project and having less subfolder will help to do so